aszczachor/src/components/mainPage/MainContent.js

```diff
@@ -0,0 +1,64 @@
+import "components/mainPage/mainContent.css";
```

Comment:
Możesz usunąć i import i plik, w pliku nic nie ma, także nie ma co trzymać, chyba że style zdjęcia przeniesiesz do oddzielnego pliku, wtedy zamień na module import. 

Jak to poprawisz to merguj już to 😄